cypress/utils/plugins/security-analytics-dashboards-plugin/index.js

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// Import commands.js using ES2015 syntax:
+import './commands';
+import 'cypress-real-events';
+
+// Alternatively you can use CommonJS syntax:
+// require('./commands')
+
+const resizeObserverLoopErrRe = /^[^(ResizeObserver loop limit exceeded)]/;
+Cypress.on('uncaught:exception', (err) => {
```

Comment:
I think this might be repetitive 

`cypress/support/index.js`

Reply:
Yes, removed it